src/api/mentions.js

```diff
@@ -79,4 +99,4 @@ const reply = (event) => {
 	}
 };
 
-module.exports = (replyMention, retweetMention);
+module.exports = (replyMention, retweetMention, getMentions);
```

Comment:
you can export as object or array, then easily import all in one by destructuring
```
module.exports = [replyMention, retweetMention, getMentions];
---
// in another file
const [replyMention, retweetMention, getMentions] = require('./src/api/mentions');
```